.github/workflows/linux-eic-shell.yml

```diff
@@ -779,7 +779,8 @@ jobs:
     - name: Download GitHub Pages staging artifact
       uses: actions/download-artifact@v4
       with:
-        path: artifacts/
```

Comment:
I think this is the wrong direction of this change.  See links in README.md (which becomes https://epic-eic.org/epic/index.html), and `reports/artifacts.md`.

I think what we may want to do is keep artifacts, and then publish _site or so.

Reply:
And we've been advertising links like https://eic.github.io/epic//artifacts/epic_craterlake_no_bhcal.stp/epic_craterlake_no_bhcal.stp to others, like project and similar gdml files to Makoto Asai. We should try not to have them break.

Reply:
Maybe we should just revert #856 and make it work first.